src/commands/infra/validate.ts

```diff
@@ -4,7 +4,10 @@ import emoji from "node-emoji";
 import shelljs from "shelljs";
```

Comment:
The running of `scaffold` also runs a `validate`.
Should we continue to ignore this error from `config.ts` : `Env variable needs to be defined for test_name`
```
naterose@naterose-Blade:~/Desktop/Microsoft/fy20/nate_fork/spk/dist$ ./spk-linux infra scaffold -n "test" --source https://infra_account:rhmnbenwc4koksstzd5rzpmpi3v37to4esmmxpbaktbg5dchwkma@naros32.visualstudio.com/spk-testing/_git/discovery-tf-template --version v0.0.1 --template "template"
error:   Env variable needs to be defined for test_name
warn:    Error loading SPK configuration file; run `spk init` to initialize your global configuration or ensure you have passed all required parameters to the called function.
warn:    The infrastructure repository containing the remote terraform template repo and access token was not specified. Checking passed arguments.
info:    All required options are configured via command line for scaffolding, expecting public remote repository for terraform templates or PAT embedded in source URL.
warn:    Converted to: _spk_testing__git_discovery_tf_template
info:    Checking if source: /home/naterose/.spk/templates/_spk_testing__git_discovery_tf_template is stored locally.
info:    Source template folder found. Validating existence of repository.
info:    Remote source repo: https://naros32.visualstudio.com/spk-testing/_git/discovery-tf-template exists.
info:    Checking if source repo: https://naros32.visualstudio.com/spk-testing/_git/discovery-tf-template has been already cloned to: /home/naterose/.spk/templates/_spk_testing__git_discovery_tf_template.
info:    https://naros32.visualstudio.com/spk-testing/_git/discovery-tf-template already cloned. Performing 'git pull'...
info:    Checking out template version: v0.0.1
info:    Terraform template files copied from /home/naterose/.spk/templates/_spk_testing__git_discovery_tf_template/template
info:    Terraform variables.tf file found. Attempting to generate definition.yaml file.
info:    A remote backend configuration was found : test/backend.tfvars
info:    A variables.tf file found : test/variables.tf
info:    --end log: 0 --
```

Reply:
I do not know. please let me know what should we do here

Reply:
@yradsmikham & @andrebriggs

Reply:
I know that environment variable is required for introspection. We should figure out a way to silent those errors if it does not concern infra.

Reply:
I can work on this if someone can tell me what to do. Create a task and assign to me would be nice :-)

Reply:
This issue appear to trying to load what I see in a mock version of `spk.config` [here](https://github.com/CatalystCode/spk/blob/ba51da30474db84d51a1ee1bea13e82c081a3114/src/commands/mocks/spk-config.yaml#L22). @samiyaakhtar is the main path code calling this config file?

@NathanielRose can you please share your `spk.config` file. In Any case let's deal with this in a separate issue.

Reply:
Posted here - https://github.com/microsoft/bedrock/issues/936